db/migrate/20160920144053_change_profile_to_json.rb

```diff
@@ -0,0 +1,45 @@
+class ChangeProfileToJson < ActiveRecord::Migration
+  class Profile < ActiveRecord::Base
+  end
+
+  def up
+    add_column :profiles, :encrypted_pii, :text
```

Comment:
because the JSON string is stored encrypted (a text string) rather than as JSON.